### PR TITLE
Syntax coloring support

### LINKED ISSUE #421: Syntax coloring support
**Is your feature request related to a problem?**

I'm always frustrated when the current plugin requires Bitlang for coloring.

**Describe the solution you'd like**

The plugin itself must be able to support coloring.

**Describe alternatives you've considered**

**Additional information**

### PR DESCRIPTION
**Pull request by grianbrcom**
Closes #421

Signed-off-by: Anton Grigorev <anton.grigorev@broadcom.com>

### CODE REVIEW

File: clients/cobol-lsp-vscode-extension/package.json [hunk 1]
```diff
@@ -100,6 +100,16 @@
                     "uniqueItems": true
                 }
             }
+        },
+        "configurationDefaults": {
+            "[COBOL]": {
+                "editor.renderIndentGuides": false,
+                "editor.rulers": [
+                    6,
+                    7,
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/package.json:109]: Should we add also 11 here to show Area B?

Comment (grianbrcom) [reply clients/cobol-lsp-vscode-extension/package.json:109]: Make sense. I could also propose 80 for showing the end of the line.
Do you think that we need it as well?

Comment (temanbrcom) [reply clients/cobol-lsp-vscode-extension/package.json:109]: It is a UX question, we should agree it with Lukas

**Comment by zimlu02** [comment]
> I hope the README for the extension is going to say where you got this from?

Hello, we plan on following legal advise and list this in the Notice file as per Eclipse's legal standards.

**Pull request merged**